src/esp32/entry.rs

```diff
@@ -76,8 +77,14 @@ pub fn serve_web(
                     .unwrap(),
             ),
             robot,
+            robot_cfg.as_ref().config.as_ref().unwrap().clone(),
         )
     };
 
+    let r = handle_webhook(robot_cfg); // TODO: add retry logic
```

Comment:
let's move that in the upper block so it's clear we drop both the webhook & the config